content/privacy.en.md

```diff
@@ -196,29 +191,67 @@ Die Weitergabe der Daten ist zur Erfüllung von Verträgen oder für vorvertragl
 The transfer of data is mandatory for the fulfilment of contracts or for pre-contractual measures. Consequently, there is no possibility of objection on the part of the user.
 
 
-<h2 id="rights">8. Rights of the Affected Person</h2>
+## 8. Using the Cryptomator Application {#app}
+
+### 8.1 Description and Scope of Data Processing
+When using one of the Cryptomator applications for desktop operating systems, iOS or Android, Data can be processed by us. This is the case when using one of the following functions:
+
+**Update-Check (Desktop, Android):** If the user checks for updates, either manually or at a scheduled interval, the application will connect to a server to query the latest version. The following data is transmitted during this request:
+
+* Operating system version
+* Software version
+* Time of the request
+* IP address
+
+**Sending of Error Protocols:** Locally created log file can help identifying problems. If the user decides to send one of these files via E-Mail, it may contain the following data:
```

Comment:
```suggestion
**Sending of Error Protocols:** Locally created log file can help identifying problems. If the user decides to send one of these files via email, it may contain the following data:
```